City-of-Helsinki/helsinki-paatokset
# PP-230: Add frontend components for news integration

Added frontend components to news integration.

To test:

- Checkout branch. Run `drush cr && drush-cim`.
- Run the news migration `drush mim paatokset_news`.
- Go to your frontpage node or create new landing page. Add _News liftups_  paragraph to your content.
- Save and view your node. You should see an element consisting of the news you just imported. Main liftup should have an image, as long as there is at least one article with an image in the description field here: https://www.hel.fi/wps/wcmjspv2/filtered-news-listing.jsp?rss=rss&categories=helsinkishared%2FPaatokset
- Click an article. This should take you to the article page. Single article should look identical to a basic HDBT article.
